middleware.go

```diff
@@ -138,6 +142,12 @@ func SetupMiddleware(api *rest.Api, mwtype string, authorizer authz.Authorizer)
 		},
 	})
 
+	// TODO: remove below once user authentication is fully implemented
+	if mwtype == EnvNoAuth {
+		// do not use `AuthzMiddleware` and return immediately instead
```

Comment:
add a huge warning that it's running without API authorization and should not be used in production